seed_services_client/service_rating.py

```diff
@@ -0,0 +1,88 @@
+
+"""
+Client for Service Rating Store HTTP services APIs.
+
+"""
+import requests
+import json
+
+
+class ServiceRatingApiClient(object):
+
+    """
+    Client for ServiceRating API.
+
+    :param str api_token:
+
+        An API Token.
+
+    :param str api_url:
+        The full URL of the API. Defaults to
+        ``http://seed-service-rating/api/v1``.
+
+    """
+
+    def __init__(self, api_token, api_url=None, session=None):
+        if api_url is None:
+            api_url = "http://seed-service-rating/api/v1"
+        self.api_url = api_url
+        self.headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'Token %s' % api_token
+        }
+        if session is None:
+            session = requests.Session()
+        session.headers.update(self.headers)
+        self.session = session
+
+    def call(self, endpoint, method, obj=None, params=None, data=None):
+        if obj is None:
+            url = '%s/%s/' % (self.api_url.rstrip('/'), endpoint)
+        else:
+            url = '%s/%s/%s/' % (self.api_url.rstrip('/'), endpoint, obj)
+        result = {
+            'get': self.session.get,
+            'post': self.session.post,
+            'patch': self.session.patch,
+            'delete': self.session.delete,
+        }.get(method, None)(url, params=params, data=json.dumps(data))
+        result.raise_for_status()
+        if method is "delete":  # DELETE returns blank body
+            return {"success": True}
+        else:
+            return result.json()
```

Comment:
@rudi Hmm, I understand what you mean, it could be used in the other clients, but at the moment it's only used in `scheduler.py`, which was copied from an older client we had in seed-stage-based-messaging. For consistency, perhaps I should update the code to look more like all the other clients where we do not use the `call` method.  See https://github.com/praekeltfoundation/seed-services-client/blob/develop/seed_services_client/stage_based_messaging.py, for instance.